datumaro/datumaro/util/__init__.py

```diff
@@ -59,4 +59,13 @@ def to_snake_case(s):
             name.append(char.lower())
         else:
             name.append(char)
-    return ''.join(name)
\ No newline at end of file
+    return ''.join(name)
+
+def str_to_bool(s):
+    t = s.lower()
+    if t in {'true', '1', 'ok', 'yes'}:
```

Comment:
Would you like to add `y` and `n` in these sets?

Reply:
Though about it, probably it is a good idea.